ts/screens/onboarding/EmailReadScreen.tsx

```diff
@@ -82,68 +108,95 @@ export class EmailReadScreen extends React.PureComponent<Props> {
       .map(_ => untag(_.spid_email))
       .getOrElse("");
 
+    const isFromProfileSection =
+      this.props.navigation.getParam("isFromProfileSection") || false;
+
+    const footerProps1: SingleButton = {
+      type: "SingleButton",
+      leftButton: {
+        bordered: true,
+        onPress: () =>
+          this.props.navigateToEmailInsertScreen(isFromProfileSection),
+        title: I18n.t("email.edit.cta")
+      }
+    };
+
+    const footerProps2: TwoButtonsInlineHalf = {
```

Comment:
When we show footer with 2 buttons the left button title seems trunked
<img src="https://user-images.githubusercontent.com/822471/67792093-dcdcfb00-fa78-11e9-8949-58ea075ff454.png" height="600" />
Maybe _Modifica_ could be just enough

Reply:
Testing it on SE, it seems the string "Edid email" is wholly displayed also on iPhone SE. I remove "address" within https://github.com/teamdigitale/io-app/pull/1363/commits/72d80b8de3378b713a0eb6c11d19690aad60667b
<img width="249" alt="image" src="https://user-images.githubusercontent.com/38431762/67849179-67b80700-fb06-11e9-982c-1c6c1401dd6a.png">

(we can see here the io-envelope icon is displayed again)